.gitignore

```
@@ -0,0 +1 @@
+test.md
```

- [ ] You've done really well to add test.md in the .gitignore file. however, as part of the requirements, you must also include the node_modules folder. this is to avoid git from tacking your node_modules folder and all the npm packages inside your actual repository.
Something like this would be fine. after this, you should be fine 👍. please kindly add this. 
```suggestion
test.md
node_modules/
```